### ISSUE TITLE
 Functionality with Microsoft Word spellchecker 

### ISSUE THREAD

**Issue by luke-scholey** [nvaccess/nvda#5797]
I have noticed that nvda doesn't read out a miss spelled word in the  spelling checker tool of Microsoft Word 2013 and 2016.

**Comment by derekriemer** [comment]
Confirmed this issue. Eight workaround for now. Using object navigation moved left until you get to the far end of the toolbar and it should read the word although it won't spell it. Until this issue is fixed this is a viable work around that you can use to access what you need.
Technically this issue is caused because word 2013 and 2016 use a spellchecker toolbar. In order to get it to read out the word it will probably have to script something up. Pressing F6 confirms that this is a toolbar (besides the fact NVDA confirms it is a toolbar already). This allows you to move back-and-forth between different parts of the window.

Sent from my heavily encrypted iPhone.
Please disregard errors as this is a smaller device.

**Comment by jcsteh** [comment]
Duplicate of #3410.
